DAOS-Support.md

```diff
@@ -98,7 +100,7 @@ $ mpirun -np 3 dcp -v /tmp/$USER/conts/p1cont1 /tmp/$USER/d
 [2020-04-23T17:17:51]   Files: 3
 [2020-04-23T17:17:51]   Links: 0
 ```
-## DAOS non-POSIX Data Movement Examples 
+## DAOS non-POSIX Data Movement Examples With dsync
 
```

Comment:
@daltonbohning just a minor typo: "With" vs. "with"

Reply:
@dsikich I don't mind either way. _**Technically**_, I think different style guides (MLA, APA, Chicago, etc.) have different rules about capitalizing "with" in a title. I'll go with the lowercase version as you suggest. (And to be consistent with other usage)

Reply:
@daltonbohning yeah, not a big deal I guess I'm just used to it being lowercase.